go/mysql/query_benchmark_test.go

```diff
@@ -29,6 +29,7 @@ import (
 var testReadConnBufferSize = connBufferSize
 
 func init() {
+	// Don't see a clear way to use pflag for this go test -benchmem flag
 	flag.IntVar(&testReadConnBufferSize, "test.read_conn_buffer_size", connBufferSize, "buffer size for reads from connections in tests")
```

Comment:
I don't see this flag being used on `arewefastyet` or anywhere else in Vitess. It was introduced in https://github.com/vitessio/vitess/pull/4196.
I think we can just switch to the `pflag` package and not do anything else i.e. just change `flag.IntVar` to `pflag.IntVar`.
This flag doesn't show up anywhere in any of the binaries. Only used when running the benchmarks in this file.

Reply:
WDYT @mattlord?